### DISCUSSION TITLE
[Theme] Allure - A more beautiful and simple theme for trilium

### DISCUSSION THREAD

**Discussion by JadeVane** [TriliumNext/Trilium#4877] [Show and tell]
Hello, everyone!

I had made a theme for trilium, and release it to github for a while, this clean theme is adopted by many people, and now, I would be honored to recommend [the Allure Theme](https://github.com/JadeVane/Allure) to you.

Allure is a theme for trilium with a more beautiful interface and higher focus which means that just focus on what you need.

## Features

- more minimalist interface
- more comfortable color scheme
- clearer layout

## Screenshot

Allure theme has made some interface element changes to provide a better-looking interface

![图片](https://github.com/user-attachments/assets/65477d78-d4e2-4df8-9c61-3213ab8d9e23)

**Comment by Nriver** [comment]
Added to https://github.com/Nriver/awesome-trilium
